src/shared/typings/core.ts

```diff
@@ -0,0 +1,3 @@
+export type ErrorDto = {
```

Comment:
Good point, got used to the `shared` folder meaning. Should we not move the types to shared instead?

Reply:
can also, no difference. but should probably do in a separate PR because nearly every file in the backend will have to update imports haha

Reply:
Maybe don't fix it here still, since types are more for backend, and shared is more for both frontend and backend it seems.

Reply:
sure, but in that case can we use `types` instead of `typings` for consistency?

Reply:
I'll move it into `types/api` instead, so we can use types across both frontend and backend as if it was shared instead of fragmenting the places where they exist

Reply:
Done in  dc95555